### PR TITLE
update(release): fix release script to update package.json files

### PR DESCRIPTION
**Pull request by ThomasBurleson**
*  Fix incorrect version num(s) in package.json files (in current master branch).
*  update release script to update package.json version # in the **master** branch

### CODE REVIEW

**Comment by ThomasBurleson** [comment]
@kara - can you review and merge if lgty.

File: release [hunk 1]
```diff
@@ -315,19 +318,27 @@
...
-      fs.writeFileSync('./package.json', JSON.stringify(json, null, 2));
+    /**
+     * When updating the 'master' branch, update both the source repository 'package.json',
+     * and also the npm version that is used when deploying the releases to npm.
+     */
+    function updatePackageVersions () {
+      [
+        './package.json',
+        './tools/scripts/release/npm_assets/package.json'
+      ].forEach(filePath => {
```
Comment (kara) [inline release]: Nit: this looks pretty similar to what you're doing in lines 109-118. Any chance you could share some code?

Comment (ThomasBurleson) [inline release]: I consider that... except `updatePackageVersions` is actually stringified to be *evaluated* in a sub node process. I worried about try it...

Comment (ThomasBurleson) [inline release]: Done.

**Pull request merged**